source/gra6-uni6/gra6-uni6-secA-lec2-cool.ptx

```diff
@@ -0,0 +1,45 @@
+<?xml version='1.0' encoding='utf-8'?>
+
+<!-- ============================================ 
+  El cool-down es un <project> 
+  (distinto a las otras actividades y a los Practice Problems)
+=================================================  -->
+<project xmlns:xi="http://www.w3.org/2001/XInclude"> 
+
+<title>¿Cómo sabes que uno solución es una solución?</title>
+
+<statement>
+  <p>
+    Explica cómo sabes que <m>88</m> es una solución a la ecuación <m>\frac{1}{8}x = 11</m> completando las frases:
+  </p>
+
+  <ul>
+    <li><p><q>La palabra "solución" significa...</q></p></li>
+    <li><p><q><m>88</m> es una solución para <m>\frac{1}{8}x = 11</m> porque...</q></p></li>
+  </ul>
+</statement>
+
+<solution>
+  <p>La palabra "solución" significa un valor que hace que la ecuación sea verdadera.</p>
+  <p><m>88</m> es una solución para <m>\frac{1}{8}x = 11</m>, porque si <m>x</m> es <m>88</m>, la ecuación es <m>\frac{1}{8} \boldcdot 88 = 11</m>, lo cual es verdadero.</p>
+</solution>
+
+
+<postlude component="profesor"> <!-- Notas para el profesor -->
+
+  <!-- Posibles errores -->
+  <!-- <paragraphs>
+    <title><custom ref="errores-cool-titulo"/></title>
+    <p>[@@@@@@@@@]</p>
+  </paragraphs> -->
+
+
+  <!-- Estrategia de apoyo si muchos estudiantes respondieron incorrectamente -->
+  <!--  <paragraphs>
+    <title><custom ref="apoyo-cool-titulo"/></title>
+    <p>[@@@@@@@@@]</p>
+  </paragraphs> -->
```

Comment:
Me parece que falta esto. La guía tiene una parte llamada [Responding to Student Thinking]

Reply:
Añadido!